Packs/CloudflareZeroTrust/Integrations/CloudflareZeroTrustEventCollector/CloudflareZeroTrustEventCollector_description.md

```diff
@@ -0,0 +1,21 @@
+## Cloudflare Zero Trust
+
+_____
+Use this integration to fetch account audit logs, user audit logs, and access authentication logs from Cloudflare Zero Trust as events in Cortex XSIAM.
+_____
+
+#### **1. Retrieve Global API Key**
+
+- Visit [Retrieve Global API Key](https://developers.cloudflare.com/fundamentals/api/get-started/keys/).
+
+  **Note:** The Global API Key is an all-purpose token that can read and edit any data or settings that you can access in the dashboard.  
+  - API Key is available here: [Cloudflare API Tokens](https://dash.cloudflare.com/profile/api-tokens)  
```

Comment:
Is the API Key different than the Global API Key?
I find this very confusing.
Why do I need to go here?